doc/source/conf.py

```diff
@@ -69,7 +69,6 @@
     "sphinx.ext.autodoc",
     "sphinx.ext.autosummary",
     "sphinx.ext.linkcode",  # This adds the button ``[Source]`` to each Python API site by calling ``linkcode_resolve``
-    "sphinx.ext.doctest",
```

Comment:
I am concerned that removing this line will not ensure the accuracy of the examples. I am trying to figure out how to address this issue. (May be able to handle this with other PRs.)

Reply:
> since we use pytest doctest-modules

I did not read this text. I'm a goofball.